src/channelape/controller/ChannelApeActionsController.ts

```diff
@@ -10,7 +10,7 @@ const PARSE_INT_RADIX = 10;
 
 export default abstract class ChannelApeActionsController {
   protected logger: Logger;
-  private updateHealthCheckInterval?: NodeJS.Timer;
+  private updateHealthCheckIntervals: {[key: string]: NodeJS.Timer} = {};
```

Comment:
I'm still a little weary of maintaining state within the controller like this. Looks like this should work to handle multiple actions at the same time but I still think the cleanest / safest solution would be to remove this field level variable and just set the interval as a function scoped variable within `handle`

Sine we are instantiating these controllers within `index.ts` within all of our web services, they are essentially singletons. We should be very weary of trying to maintain state on a singleton.

Reply:
@rjdavis3  Actually should we wait to finish CA-4318 and use the version that comes out of this PR in that repo?

Reply:
@csimko I am not too worried about CA-4318 since we only use the error reporting functionality in that repo. These changes will not impact that. This will impact all of our *-web-services.